content/zotero_intro_to.md

```diff
@@ -458,8 +458,7 @@ Produce a document identical to this text.
 * Retrieve and add to your Zotero library the bibliographic references cited, you should find all of them online.
 * Make sure you have installed APA style.
 
-**|!| Tips**
-
+**💡 Tips**
```

Comment:
to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

Reply:
Ok